src/Integrations/Integrations/CakePHP/CakePHPIntegration.php

```diff
@@ -72,8 +72,10 @@ function (SpanData $span, array $args) use ($integration) {
                     return;
                 }
 
-                $integration->rootSpan->resource =
-                    $_SERVER['REQUEST_METHOD'] . ' ' . $this->name . 'Controller@' . $request->params['action'];
+                if (dd_trace_env_config("DD_HTTP_SERVER_ROUTE_BASED_NAMING")) {
```

Comment:
Don't we want to reuse `DD_TRACE_URL_AS_RESOURCE_NAMES_ENABLED` in that case as well instead of introducing a new param?

Reply:
No we don't :-) That'd give a ternary overload to `DD_TRACE_URL_AS_RESOURCE_NAMES_ENABLED`.